src/View/SSViewer_Scope.php

```diff
@@ -172,58 +171,21 @@ public function locally()
     }
 
     /**
-     * Reset the local scope - restores saved state to the "global" item stack. Typically called after
-     * a lookup chain has been completed
+     * Set scope to an intermediate value, which will be used for getting output later on.
      */
-    public function resetLocalScope()
```

Comment:
`resetLocalScope` still exists, it's just been moved lower and made protected. Nothing outside this class hierarchy needs to touch it.